.github/workflows/fleek-deploy.yaml

```diff
@@ -0,0 +1,42 @@
+name: Deploy site via Fleek
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Should we do a separate deployment for main? I'm assuming the site ID in fleek.json is for the dev site.

Reply:
ah, yeah! will separate it